cmd/object-api-utils.go

```diff
@@ -825,3 +825,23 @@ func (d *detectDisconnect) Read(p []byte) (int, error) {
 		return d.ReadCloser.Read(p)
 	}
 }
+
+// Return the sum of the values in a map
+func sumMapValues(m map[string]int) (sum int) {
```

Comment:
It is better to come up with some struct here such that you can simply do `m.Sum()` to get the actual sum here. and then implement the reduceMaps as `m1.Merge(m2)`  - would be more idiomatic Go style.